src/storybook/story-test-runner/index.ts

```diff
@@ -0,0 +1,58 @@
+import { extendStoriesFromStoryFile } from "./extend-stories";
+import { nestedDescribe, extendedIt } from "./test-decorators";
+import { openStory } from "./open-story";
+import type { StoryLoadResult } from "./open-story/hermione-open-story";
+import type { HermioneOpts } from "../story-to-test";
+import type { TestFunctionExtendedCtx } from "../../types";
+import type { StorybookStoryExtended, StorybookStory } from "./types";
+
+export function getAbsoluteFilePath(): string {
+    return __filename;
+}
+
+export function run(stories: StorybookStory[], opts: HermioneOpts): void {
+    const withStoryFileDataStories = extendStoriesFromStoryFile(stories);
+
+    withStoryFileDataStories.forEach(story => createHermioneTests(story, opts));
+}
+
+function createHermioneTests(story: StorybookStoryExtended, { autoScreenshots }: HermioneOpts): void {
+    nestedDescribe(story, () => {
+        if (autoScreenshots) {
+            extendedIt(story, "Autoscreenshot", async function (ctx: TestFunctionExtendedCtx) {
+                ctx.expect = globalThis.expect;
```

Comment:
Storybook has his own `expect`.

Putting our `expect` to context, so those would not be missmatched, and ts typings would work for both.